### PR TITLE
vokoscreen-ng: 3.0.8 -> 3.5.0

### PR DESCRIPTION
**Pull request by leiserfg**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/vokoscreen-ng/default.nix [hunk 1]
```diff
@@ -57,7 +53,12 @@ mkDerivation rec {
   '';
 
   postInstall = ''
+    mkdir -p $out/bin $out/share/applications $out/share/icons
+    cp ./vokoscreenNG $out/bin/
+    cp ./src/applications/vokoscreenNG.desktop $out/share/applications/
+    cp ./src/applications/vokoscreenNG.png $out/share/icons/
     qtWrapperArgs+=(--prefix GST_PLUGIN_SYSTEM_PATH_1_0 : "$GST_PLUGIN_SYSTEM_PATH_1_0")
+    wrapQtApp $out/bin/vokoscreenNG
```
Comment (SuperSandro2000) [inline pkgs/applications/video/vokoscreen-ng/default.nix:61]: Is this now double wrapping?

File: pkgs/applications/video/vokoscreen-ng/default.nix [hunk 2]
```diff
@@ -15,36 +15,32 @@
 , gst-plugins-good
 , gst-plugins-bad
 , gst-plugins-ugly
+, wayland
+, wrapQtAppsHook
 }:
 mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/applications/video/vokoscreen-ng/default.nix:22]:
```suggestion
}:

mkDerivation rec {
```

File: pkgs/applications/video/vokoscreen-ng/default.nix (hunk 1 above)
Comment (leiserfg) [reply pkgs/applications/video/vokoscreen-ng/default.nix:61]: No, if I don't do it manually  it's not detecting the binary not sure why (I even had to copy the binary by hand).
It screams a 
```sh
/bin: Is a directory
make: Nothing to be done for 'install'.
```

**Comment by leiserfg** [comment]
Do I need to change something else?

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by ilya-fedin** [comment]
Is this a missing buildInputs dependency?
изображение

**Comment by leiserfg** [comment]
It could be, seems like it is a Wayland only dynamic dependency and I only tested it in x11

**Comment by ilya-fedin** [comment]
Adding pipewire (which contains the gstreamer plugin) to buildInputs helped to me

**Comment by SuperSandro2000** [comment]
Please create a PR tot fix that for everyone

**Pull request merged**